src/app/main/component/user/components/profile/profile-cards/profile-cards.component.html

```diff
@@ -6,4 +6,12 @@
       <img src="assets/img/wave_shape.png" alt="wave-background" />
     </div>
   </div>
+
+  <div class="card habit-card" *ngIf="habitFactOfTheDay">
+    <p class="cart-title">{{ 'profile.fact-of-the-day' | translate }}</p>
+    <p class="card-description">{{ habitFactOfTheDay?.content }}</p>
+    <div class="shape-img">
+      <img src="assets/img/wave_shape.png" alt="wave-background" />
```

Comment:
You can put the image path in `src/app/main/image-pathes/profile-`
Use ngSrc (NgOptimizedImage directive) to improve performance